addons/website/tests/test_snippets.py

```diff
@@ -29,8 +48,8 @@ def test_03_snippets_all_drag_and_drop(self):
         snippets_names = ','.join(set(el.attrib['data-snippet'] for el in data_snippet_els if el.attrib['data-snippet'] not in blacklist))
         self.start_tour("/?enable_editor=1&snippets_names=%s" % snippets_names, "snippets_all_drag_and_drop", login='admin', timeout=300)
 
-    def test_04_countdown_preview(self):
+    def test_05_countdown_preview(self):
```

Comment:
I was about to give this PR another round of manual testing to make sure we do not introduce any deadlock but while I'm here:
Wouldn't it be better if you just moved your test at the end of the file? So that you do not touch lines that are not related to your fix 😄

Reply:
I'd say it's probably fine that your test's name changes between 14.0 and 15.0. It has some drawbacks mainly that it won't be as easy to find the test in past versions.

But for some test, their signature is the easy access to the blame / commit that introduced them, so it's even easier to see what they were supposed to ensure is working. For example:
If you change the definition of the test_04_countdown_preview method in 16.0+ you will remove any reference to the commit that introduced the test from the inline blame, you will have to go back one revision further which isn't ideal. IMO that's a bigger drawback that the name change, but your opinion is as valid as mine 👍🏻

Reply:
> If you change the definition of the test_04_countdown_preview method in 16.0+ you will remove any reference to the commit that introduced the test from the inline blame, you will have to go back one revision further which isn't ideal. IMO that's a bigger drawback that the name change

I agree with you! This has been adapted :+1: